compiler/src/iree/compiler/Codegen/Common/FoldTensorSubsetIntoVectorTransferOps.cpp

```diff
@@ -268,25 +268,26 @@ class FoldExtractSliceIntoTransferWrite final
     auto init = rewriter.create<tensor::EmptyOp>(
         loc, mixedSizes, extractSliceOp.getType().getElementType());
 
+    auto indices = xferOp.getIndices();
+
     SmallVector<bool> inBounds;
     inBounds.resize(mixedSizes.size());
```

Comment:
How about initializing all of them to false in constructor, instead of setting values in the for loop?

```suggestion
    SmallVector<bool> inBounds(mixedSizes.size(), false);
```

Reply:
sure. done.